src/beeware_docs_tools/build_md_translations.py

```diff
@@ -69,29 +68,27 @@ def generate_translated_md(
     )
 
 
-def build_docs(config_file: Path, output_path: Path) -> None:
+def build_docs(config_file: Path, output_path: Path, build_with_warnings: bool) -> None:
     """
     Run `mkdocs build` for a given language.
 
     :param config_file: The path to the language-specific configuration file.
     :param output_path: The output directory for the site content.
     """
-    args = parse_args()
-
     build_command = [
         "python",
         "-m",
         "mkdocs",
         "build",
         "--clean",
         "--config-file",
-        f"{config_file}",
+        str(config_file),
         "--site-dir",
-        f"{output_path}",
+        str(output_path),
     ]
 
-    if not args.build_with_warnings:
-        build_command.extend(["--strict"])
+    if not build_with_warnings:
+        build_command.append("--strict")
```

Comment:
I feel like there was a reason I had to use `extend`, but I don't recall what it was. Can you please explain why you changed this?

Reply:
Passing a list of one item to `extend()`, and passing a single item to `append()`, are functionally the same. The difference between the two is whether you're passing in a list (which *could* have more than one item in it). `extend` requires a list; `append` can only ever accept a single value.